packages/components/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@backstage/components",
+  "description": "Components used by Backstage plugins and apps",
+  "version": "0.1.1-alpha.20",
+  "private": false,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/spotify/backstage",
+    "directory": "packages/components"
+  },
+  "keywords": [
+    "backstage"
+  ],
+  "license": "Apache-2.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "scripts": {
+    "build": "backstage-cli build --outputs types,esm",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/config": "^0.1.1-alpha.20",
+    "@backstage/core": "^0.1.1-alpha.20",
```

Comment:
Shouldn't be any need to depend on core here, since core just ends up being a thin glue around `core-api` and `core-components`